tests/test_irsb.py

```diff
@@ -1,39 +1,40 @@
-import angr
-import pyvex
-import archinfo
+import unittest
 
 import ailment
 
-block_bytes = bytes.fromhex("554889E54883EC40897DCC488975C048C745F89508400048C745F0B6064000488B45C04883C008488B00BEA70840004889C7E883FEFFFF")
-block_addr = 0x4006c6
+try:
+    import angr
+    import archinfo
+except:
+    angr = None
 
-def test_convert_from_vex_irsb():
+try:
+    import pyvex
+except:
+    pyvex = None
 
-    arch = archinfo.arch_from_id('AMD64')
 
-    manager = ailment.Manager(arch=arch)
+@unittest.skipUnless(angr, "angr required")
```

Comment:
I believe the first test case test_convert_from_vex_irsb only requires pyvex, not the entire angr.